imports/ui/components/buttons/backButton.js

```diff
@@ -1,9 +1,10 @@
 import { FlowRouter } from 'meteor/kadira:flow-router'
+import { setIsNavigatingBack, getPreviousPath, removeLastPreviousPath } from '/imports/lib/utils'
 import './backButton.html'
 
 const getPrevPage = () => (
   Template.instance().data.prevPage ||
```

Comment:
Is this still needed? What does it do?

Reply:
This is in fact still needed (unless we want to change the current behavior). Certain pages should always "go back" to a specific page, not necessarily the last one in the history stack. Our only example of this now is going from a playlist back to the list of playlists (I think @pedrocasa can confirm that this is the behavior we wanted). To specify a hard-coded "previous route", we pass it into the template and this controller treats it as an override to the actual history.